server/src/main/webapp/token/index.html

```diff
@@ -0,0 +1,65 @@
+<!--
+  ~  Copyright (c) 2009 - ${YEAR} Red Hat, Inc.
+  ~  
+  ~  This software is licensed to you under the GNU General Public License,
+  ~  version 2 (GPLv2). There is NO WARRANTY for this software, express or
+  ~  implied, including the implied warranties of MERCHANTABILITY or FITNESS
+  ~  FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+  ~  along with this software; if not, see
+  ~  http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+  ~  
+  ~  Red Hat trademarks are not licensed under GPLv2. No permission is
+  ~  granted to use or replicate Red Hat trademarks that are incorporated
+  ~  in this software or its documentation.
+  -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="utf-8"/>
+</head>
+<body>
+<script src="keycloak.js"></script>
+<script>
+        var status;
+        var xhttp = new XMLHttpRequest();
+        xhttp.open('GET', "https://localhost:8443/candlepin/status", true);
```

Comment:
This cannot be 'localhost', because it will not work in production. You something like this:

```javascript
var host = window.location.hostname;
xhttp.open('GET', "https://" + host + "8443/candlepin/status", true);
```